vci-service-impl/src/main/java/io/mosip/certify/vci/filter/AccessTokenValidationFilter.java

```diff
@@ -0,0 +1,107 @@
+/*
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/.
+ */
+package io.mosip.certify.vci.filter;
+
+import io.mosip.certify.core.dto.vci.ParsedAccessToken;
+import io.mosip.certify.core.util.IdentityProviderUtil;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.security.oauth2.core.DelegatingOAuth2TokenValidator;
+import org.springframework.security.oauth2.jwt.*;
+import org.springframework.stereotype.Component;
+import org.springframework.web.filter.OncePerRequestFilter;
+
+import jakarta.servlet.*;
+import jakarta.servlet.http.HttpServletRequest;
+import jakarta.servlet.http.HttpServletResponse;
+import java.io.IOException;
+import java.time.Clock;
+import java.time.Instant;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Objects;
+
+import static io.mosip.certify.core.spi.TokenService.CLIENT_ID;
+
+@Slf4j
+@Component
+public class AccessTokenValidationFilter extends OncePerRequestFilter {
+
+    @Value("${mosip.esignet.vci.authn.issuer-uri}")
```

Comment:
Why are we still using "mosip.esignet" prefix?

Reply:
we are using esignet instance to get the public jwk for access token verification. So left it as is since this is a esignet reference

Reply:
Since it is currently change to mosip.certify, marking this resolved